mars_api/data_import/data_importer.py

```diff
@@ -0,0 +1,145 @@
+"""Like Transporter but without Jason Statham."""
+from sys import stdout
+
+from dateutil import parser
+from django.db.utils import IntegrityError
+
+from mars_api.serializers import (GameSerializerForImportedData,
+                                  PlayerScoreSerializer)
+
+FIELD_REMAPPING = {
+    "game": {"map": "Default"},
+    "player_score": {
+        "corporation": {
+            "Interpl. Cinematics": "Interplanetary Cinematics",
+            "UNMI": "United Nations Mars Initiative",
+        }
+    },
+}
+
+
+def str_to_bool(arg):
+    return bool(int(arg))
+
+
+GAME_FIELD_MAPPING = {
+    0: {"id": int},
+    1: {"date": parser.parse},  # datetime
+    2: {"game_map": str},
+    3: {"prelude": str_to_bool},
+    4: {"colonies": str_to_bool},
+    5: {"number_of_generations": int},
+}
+
+PLAYER_SCORE_FIELD_MAPPING = {
+    7: {"player_nickname": str},
```

Comment:
This structure can be improved I think. I reckon you don't need to have a dictionary here and could use the indexes in order to recognise the data on the other end - it's expected to have the same structure, right?

Reply:
well, I used dictionary because I'm skipping the "winner" column which is the 9th index.

Reply:
Then you can skip it :P

Reply:
That would require defining a separate index variable and make things more complicated than they have to be. I feel like this mapping and your optimized `list_to_dict` is a more elegant solution